app/services/entity_update_service.rb

```diff
@@ -18,19 +18,15 @@ def self.run(entity:, params:, current_user:)
 
         entity.update_extension_records(entity_params.extension_def_ids)
 
-        if entity_params.submitted_with_regions?
```

Comment:
We want to be able to remove all the regions/countries, so don't want to skip this even if no regions/countries